### ISSUE TITLE
can not use CBDT/CBLC font 

### ISSUE THREAD

**Issue by llxiaoyuan** [ocornut/imgui#8190]
### Version/Branch of Dear ImGui:

Version 1.XX, Branch: XXX (master/docking/etc.)

### Back-ends:

imgui_impl_dx11_cpp + imgui_impl_dx11.cpp

### Compiler, OS:

win10

### Full config/build information:

I used the FREETYPE_LUNASVG flag to load TwitterColorEmoji-SVGinOT.ttf and noto-untouchedsvg.ttf, but I still cannot load AppleColorEmoji.ttf.
Can CBDT/CBLC color font format be supported? Or is there a way to convert CBDT/CBLC to SVG format?

### Details:

I used the FREETYPE_LUNASVG flag to load TwitterColorEmoji-SVGinOT.ttf and noto-untouchedsvg.ttf, but I still cannot load AppleColorEmoji.ttf.
Can CBDT/CBLC color font format be supported? Or is there a way to convert CBDT/CBLC to SVG format?

**Comment by ocornut** [comment]
Maybe @pthom would know?
Try PlutoSVG instead it seems more up to date than LunaSVG.

**Comment by pthom** [comment]
And as far as your specific font is concerned, you would need to enable PNG support in freetype. I'm not sure it will be enough, though. I failed displaying glyphs from this particular font on my side, even with png support in freetype.

**Comment by llxiaoyuan** [comment]
AddCustomRectFontGlyph is a good way？

**Comment by ocornut** [comment]
> AddCustomRectFontGlyph is a good way？

Yes. But notice this issue, you need to mark the glyph as colored:
https://github.com/ocornut/imgui/issues/8133#issuecomment-2462312266
